### ISSUE TITLE
Wrong import paths with `near-operation-file` preset (on Windows)

### ISSUE THREAD

**Issue by vhenzl** [dotansimha/graphql-code-generator#1892]
In the recent canary version, with new `near-operation-file` preset, import paths in "near" files are malformed. 

My guess is it that has something to do with Windows file separator, but I wasn't able to replicate it consistently as I got two different behaviours. Either the relative path contains backslashes: 
```js
import * as Types from '..\..\..\__generated__/types';`
```
or there aren't any slashes at all:
```js
import * as Types from "......__generated__/types";
```

Two different projects, the same machine. 

**`codegen.yml` config file:**

```yml
overwrite: true
schema: "schema.graphql"
documents: 'src/**/*.{graphql,tsx}'
generates:
  src/__generated__/types.ts:
    plugins:
      - "typescript"
  src/:
    preset: near-operation-file
    presetConfig:
      extension: .generated.ts
      baseTypesPath: __generated__/types.ts
    plugins:
      - typescript-operations
```

**Environment:**

- OS: Windows 10
- `@graphql-codegen/...@1.1.4-alpha-62ee81ed.50`: 
- NodeJS: v10.15.3

**Comment by dotansimha** [comment]
I think you are right, and it's relevant only for Windows environment.
Actually we are using `path` to do all these resolution, so maybe it's related to it?

@ardatan you have Windows machine, right? Can you please check that?

**Comment by dotansimha** [comment]
I noticed that we were using explicit `/` in some parts of the code, I tried to change it to `path.sep` so it will use the correct separator according to the environment. Fixed it in https://github.com/dotansimha/graphql-code-generator/commit/732e71bbb1f99f29a46c6fd2476ab1ef45901ba2 
@vhenzl can you please try `1.1.4-alpha-e60d4c6c.56`?

**Comment by vhenzl** [comment]
Nah, it has to be the other way around. Now it creates import like this:

```js
import * as Types from '..\..\..\__generated__\types';
```

but what we want is:

```js
import * as Types from '../../../__generated__/types';
```

i.e. import path has to be with "normal" forward slashes regardless of OS.

**Comment by vhenzl** [comment]
It doesn't seem there is a way how to do it directly in node, it will probably require something like https://www.npmjs.com/package/slash.

**Comment by vhenzl** [comment]
@dotansimha If you agree with using `slash` package, I can try and prepare PR with a fix.

**Comment by dotansimha** [comment]
The fix is available in 1.2.0, thanks @vhenzl !
